components/shared/searchFilters/SearchFiltersColumn.tsx

```diff
@@ -0,0 +1,90 @@
+import { SearchFacetsQuery } from "@/lib/graphql/generated/fbi/graphql"
+import { cn } from "@/lib/utils"
+import React, { useEffect, useRef, useState } from "react"
+import { mapFacetsToFilters, mapFilterNameToTranslation, toggleFilter } from "./helper"
+import { useRouter, useSearchParams } from "next/navigation"
+import Icon from "../icon/Icon"
+import { Button } from "../button/Button"
+
+type SearchFiltersColumnProps = {
+  facet: SearchFacetsQuery["search"]["facets"][0]
```

Comment:
Don't we have a fragment for the facets. Otherwise I think we should create one and use that directly.